recipe/meta.yaml

```diff
@@ -10,12 +10,12 @@ source:
   sha256: 949efd451a09435f7e8573574c71c7b71b194d844890fa49cd61d2262ea1a440
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
 
 requirements:
   build:
-    - {{ compiler('c') }}
```

Comment:
We should *add* stdlib in addition to the c compiler, not replace it.

Reply:
Done with 47bbca0. 

Thanks! I see now that I had misread the guidance around this change.